packages/grafana-prometheus/src/querybuilder/components/PromQueryBuilder.test.tsx

```diff
@@ -89,7 +89,7 @@ describe('PromQueryBuilder', () => {
   it('tries to load metrics without labels', async () => {
     const { languageProvider, container } = setup();
     await openMetricSelect(container);
-    await waitFor(() => expect(languageProvider.getLabelValues).toHaveBeenCalledWith('__name__'));
+    await waitFor(() => expect((languageProvider.getLabelValues as jest.Mock).mock.calls[0][1]).toEqual('__name__'));
```

Comment:
i think we can use jest [`expect.anything()`](https://jestjs.io/docs/expect#expectanything) here to make this more readable. 

```ts
await waitFor(() => expect(languageProvider.getLabelValues).toHaveBeenCalledWith(expect.anything(), '__name__'));
```

Reply:
https://github.com/grafana/grafana/pull/101889/commits/f16a50ef7463b94f5aa6f67e949b8fd0e45d9a2b